data/human/variants.txt

```diff
@@ -1073,7 +1072,6 @@ ship "Cruiser" "Cruiser (Plasma Anti-Missile)"
 		"LP144a Battery Pack"
 
 
-
```

Comment:
3 lines is standard between variants of different ships, 2 between variants of the same ship

Reply:
I did that by accident while editing. Will revert.